### PR TITLE
PP-9214 add `organisation_details` Stripe setup task

### PR DESCRIPTION
**Pull request by stephencdaly**

### CODE REVIEW

File: src/main/java/uk/gov/pay/connector/gatewayaccount/service/StripeAccountSetupService.java [hunk 1]
```diff
@@ -46,6 +46,8 @@ public StripeAccountSetup getCompletedTasks(long gatewayAccountId) {
                         case GOVERNMENT_ENTITY_DOCUMENT:
                             stripeAccountSetup.setGovernmentEntityDocument(true);
                             break;
+                        case ORGANISATION_DETAILS:
+                            stripeAccountSetup.setOrganisationDetailsCompleted(true);
                         default:
```
Comment (SandorArpa) [inline src/main/java/uk/gov/pay/connector/gatewayaccount/service/StripeAccountSetupService.java:52]: do we need a break; ?

**Review by SandorArpa — CHANGES_REQUESTED** [review]

File: src/main/java/uk/gov/pay/connector/gatewayaccount/service/StripeAccountSetupService.java (hunk 1 above)
Comment (stephencdaly) [reply src/main/java/uk/gov/pay/connector/gatewayaccount/service/StripeAccountSetupService.java:52]: yes! good spot

**Review by SandorArpa — APPROVED** [review]
LOKTM

File: src/main/java/uk/gov/pay/connector/gatewayaccount/service/StripeAccountSetupService.java (hunk 1 above)
Comment (stephencdaly) [reply src/main/java/uk/gov/pay/connector/gatewayaccount/service/StripeAccountSetupService.java:52]: updated

**Pull request merged**